src/components/PostDetails.tsx

```diff
@@ -132,9 +80,9 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
                     type="button"
                     className="delete is-small"
                     aria-label="delete"
-                    onClick={() => deleteComment(comment.id)}
+                    onClick={() => deleteComment(comment)}
                   >
-                    delete button
+                    {/* delete button */}
```

Comment:
if u don't need this text - remove comment.
Remove all the commetns from project